automated_provisioning/kube_instance/my_instance_type.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: amlarc.azureml.com/v1alpha1
+kind: InstanceType
+metadata:
+  name: defaultinstancetype
+spec:
+  nodeSelector:
+    training_node: my_beefy_node
+  resources:
+    limits:
+      cpu: "16"
+      nvidia.com/gpu: 8
+      memory: "128Gi"
```

Comment:
Can you add a link / guidelines on what these values mean and how to choose them?